devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1811,6 +1821,10 @@ bool WholeBodyDynamicsDevice::attachAllIMUs(const PolyDriverList& p)
     }
     // We try to read for a brief moment the sensors for two reasons:
     // so we can make sure that they actually work, and to make sure that the buffers are correctly initialized
+
+    if (settings.disableSensorReadCheckAtStartup) {
+        return true;
+    }
```

Comment:
I guess the same logic should be at the end of `attachAllFTs`, what do you think?